packages/slice-machine/components/AppLayout/Navigation/index.test.tsx

```diff
@@ -0,0 +1,133 @@
+// @vitest-environment jsdom
+import { describe, test, expect, vi } from "vitest";
+import {
+  render,
+  within,
+  screen,
+  type RenderReturnType,
+} from "../../../test/__testutils__";
+import SideNavigation from "./index";
+import { FrontEndEnvironment } from "@lib/models/common/Environment";
+import { UserContextStoreType } from "@src/modules/userContext/types";
+import { act } from "react-dom/test-utils";
+
+import Router from "next/router";
+
+const mockRouter = vi.mocked(Router);
+
+vi.mock("next/router", () => import("next-router-mock"));
+
+function renderApp(): RenderReturnType {
+  return render(<SideNavigation />, {
+    preloadedState: {
+      availableCustomTypes: {},
+      slices: {
+        libraries: [
+          {
+            name: "slices/a",
+            path: "slices/a",
+            components: [],
+            isLocal: true,
+            meta: {
+              isNodeModule: false,
+              isDownloaded: false,
+              isManual: true,
+            },
+          },
+          {
+            name: "slices/b",
+            path: "slices/b",
+            components: [],
+            isLocal: true,
+            meta: {
+              isNodeModule: false,
+              isDownloaded: false,
+              isManual: true,
+            },
+          },
+        ],
+        remoteSlices: [],
+      },
+      environment: {
+        repo: "foo",
+        manifest: {
+          apiEndpoint: "https://foo.cdn.prismic.io/api/v2",
+        },
+        changelog: {
+          currentVersion: "",
+          updateAvailable: true,
+          latestNonBreakingVersion: null,
+          versions: [],
+        },
+      } as unknown as FrontEndEnvironment,
+      userContext: {
+        hasSeenTutorialsToolTip: false,
+      } as unknown as UserContextStoreType,
+    },
+  });
+}
+
+describe("Side Navigation", () => {
+  test("Logo and repo area", async () => {
+    renderApp();
+    expect(await screen.findByText("foo")).toBeVisible();
+    expect(await screen.findByText("foo.prismic.io")).toBeVisible();
+    const link = await screen.findByTitle("Open prismic repository");
+    expect(link).toHaveAttribute("href", "https://foo.prismic.io");
+    expect(link).toHaveAttribute("target", "_blank");
+  });
+
+  test("Update box", () => {
+    renderApp();
+    expect(screen.findByText("Updates Available"));
```

Comment:
We always want to expect something with toBe.... or something else.

```suggestion
    expect(screen.findByText("Updates Available")).toBeVisible();
```

Reply:
Maybe we could add the opposite case when it's not in the document?